lfucg-exactions/server/plats/models.py

```diff
@@ -47,6 +46,19 @@ class Plat(models.Model):
     is_approved = models.BooleanField(default=False)
     is_active = models.BooleanField(default=True)
 
+    EXPANSION_AREAS = (
+        ('EA-1', 'EA-1'),
+        ('EA-2A', 'EA-2A'),
+        ('EA-2B', 'EA-2B'),
+        ('EA-2C', 'EA-2C'),
+        ('EA-3', 'EA-3'),
+    )
+
+    PLAT_TYPES = (
+        ('PLAT', 'Final Record Plat'),
+        ('DEVELOPMENT_PLAN', 'Final Development Plan'),
+    )
+
```

Comment:
I am not sure how to set those lists across different model pages so we don't have circular references.